55.RGB18E7S3/main.cpp

```diff
@@ -43,15 +43,19 @@ int main()
 {
     constexpr std::string_view APP_NAME = "RGB18E7S3 utility test";
 
-	CommonAPI::InitOutput initOutput;
-    CommonAPI::InitWithNoExt(initOutput, video::EAT_VULKAN, APP_NAME.data());
+    CommonAPI::InitParams initParams;
+    initParams.apiType = video::EAT_VULKAN;
+    initParams.appName = APP_NAME.data();
+    initParams.swapchainImageUsage = nbl::asset::IImage::E_USAGE_FLAGS(0);
+    auto initOutput = CommonAPI::Init(std::move(initParams));
+
 	auto system = std::move(initOutput.system);
     auto gl = std::move(initOutput.apiConnection);
     auto logger = std::move(initOutput.logger);
     auto gpuPhysicalDevice = std::move(initOutput.physicalDevice);
     auto logicalDevice = std::move(initOutput.logicalDevice);
     auto queues = std::move(initOutput.queues);
-    auto renderpass = std::move(initOutput.renderpass);
+    auto renderpass = std::move(initOutput.renderToSwapchainRenderpass);
```

Comment:
this example is headless

Reply:
same as example 02 where an old fbo signature is seen